MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -301,6 +301,10 @@
      changes the app and the language of the speech-recognition-based search to 'Dutch'. -->
     <string name="settings_language_dutch">Dutch</string>
 
+    <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
+         changes the app and the language of the speech-recognition-based search to 'Thai'. -->
+    <string name="settings_language_thai">Thai</string>
```

Comment:
I don't think have Thai in our language settings :)
(see for a full list of languages: https://github.com/mozilla-l10n/android-l10n/blob/master/MozillaReality/FirefoxReality/l10n.toml#L5)

Reply:
We should add 'th' is our language setting, thanks for reminding me.

Reply:
I might be confused, but I don't see that there is a Thai localization for Reality: https://github.com/mozilla-l10n/android-l10n/blob/master/MozillaReality/FirefoxReality/l10n.toml#L5
Does it live elsewhere (maybe speech recognition is different)?

Reply:
We haven 't done any localization for Thai, only added its keyboard.

Reply:
We have a PR now (https://github.com/MozillaReality/FirefoxReality/pull/3579). Should we wait for your localization work?

Reply:
@Delphine The app is not yet localized in Thai yet, we just added support for the Thai keyboard. That string is displayed in the FxR keyboard chooser panel.

Reply:
So the string comment doesn't reflect what this string actually does?

Reply:
> We haven 't done any localization for Thai, only added its keyboard.

Ok comment should be updated, then. Thanks!

Reply:
It seems that we are using that string for both the keyboard name and the language entry in the UI language chooser list. Maybe we should update the comments to reflect that. @daoshengmu

Reply:
@keianhzo: Friendly reminder that we won't be able to land any strings - and expose them to localizers - until this comment is updated (mentioning this as I see new strings coming up). Thanks!

Reply:
Addessed in https://github.com/MozillaReality/FirefoxReality/pull/3633